doc/classes/Node3D.xml

```diff
@@ -232,143 +239,159 @@
+			[b]Note:[/b] This is [b]not[/b] the property available in the Inspector dock.
 		</member>
 		<member name="rotation_edit_mode" type="int" setter="set_rotation_edit_mode" getter="get_rotation_edit_mode" enum="Node3D.RotationEditMode" default="0">
-			Specify how rotation (and scale) will be presented in the editor.
+			How this node's rotation and scale are displayed in the Inspector dock.
 		</member>
 		<member name="rotation_order" type="int" setter="set_rotation_order" getter="get_rotation_order" enum="EulerOrder" default="2">
-			Specify the axis rotation order of the [member rotation] property. The final orientation is constructed by rotating the Euler angles in the order specified by this property.
+			The axis rotation order of the [member rotation] property. The final orientation is calculated by rotating around the X, Y, and Z axis in this order.
 		</member>
 		<member name="scale" type="Vector3" setter="set_scale" getter="get_scale" default="Vector3(1, 1, 1)">
-			Scale part of the local transformation.
-			[b]Note:[/b] Mixed negative scales in 3D are not decomposable from the transformation matrix. Due to the way scale is represented with transformation matrices in Godot, the scale values will either be all positive or all negative.
-			[b]Note:[/b] Not all nodes are visually scaled by the [member scale] property. For example, [Light3D]s are not visually affected by [member scale].
+			Scale of this node in parent space (relative to the parent node). This value is obtained from [member basis]'s scale.
+			[b]Note:[/b] The behavior of some 3D node types is not affected by this property. These include [Light3D], [Camera3D], [AudioStreamPlayer3D], etc.
+			[b]Warning:[/b] The scale's components must either be all positive or all negative, and [b]not[/b] exactly [code]0.0[/code]. Otherwise, it won't be possible to obtain the scale from the [member basis]. This may cause the intended scale to be lost when reloaded from disk, and potentially other unstable behavior.
 		</member>
 		<member name="top_level" type="bool" setter="set_as_top_level" getter="is_set_as_top_level" default="false">
-			If [code]true[/code], the node will not inherit its transformations from its parent. Node transformations are only in global space.
+			If [code]true[/code], the node does not inherit its transformations from its parent. As such, node transformations will only be in global space, which also means that [member global_transform] and [member transform] will be identical.
 		</member>
 		<member name="transform" type="Transform3D" setter="set_transform" getter="get_transform" default="Transform3D(1, 0, 0, 0, 1, 0, 0, 0, 1, 0, 0, 0)">
-			Local space [Transform3D] of this node, with respect to the parent node.
+			The transformation of this node, in parent space (relative to the parent node). Contains and represents this node's [member position], [member rotation], and [member scale].
```

Comment:
This drops the `[Transform3D]`.

For the transform itself, I think describing it as local space is both accurate and helpful, because people new to Godot need to immediately know that this isn't global.

There are 2 sets of transforms (all parents, and this node's transform), which result in 3 spaces (global with neither, parent-relative with just all parents, and object-local with both). For other properties and methods, it makes sense to clarify parent-relative or object-local because we need to clarify "Is it local in the same way that the node's transform is local, or is it local within the node's transform, even more local than the local transform?" but for the transform itself, there is no confusion, local can only mean one thing.

```suggestion
			Local space [Transform3D] of this node, relative to the parent node. Contains and represents this node's [member position], [member rotation], and [member scale].
```

Reply:
> This drops the `[Transform3D]`.

It's intentional. There's often no need to specify the type when it is already defined by the property header.